modules/ROOT/pages/endpoint-as2-receive.adoc

```diff
@@ -1,49 +1,67 @@
 = AS2 Receive Endpoint Settings
 
-AS2 receive endpoint settings configure the source endpoint in an inbound message flow that receives B2B messages from partners directly or via third-party connections. One AS2 endpoint can receive data from multiple partners, third-party connections, or both.
+AS2 receive endpoint settings determine where to receive B2B messages received over AS2.
 
-Only a host can own AS2 receive endpoints. Use a host-owned AS2 receive endpoint for an inbound message flow that uses a third-party connection.
+An AS2 Receive from Partners endpoint (the only type of AS2 receive endpoint) acts as the source endpoint in an inbound message flow that receives B2B messages from partners either directly or via third-party connections.
```

Comment:
Is the parenthetical info need-to-have or just nice-to-have? If the latter, consider omitting. This stuff is already dense without adding nice-to-have....